lib/shared/config/cli-flags.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var yargs = require('yargs');
+var camelCase = require('lodash.camelcase');
+
+function mergeToCliFlags(cliFlags, config, cliOptions) {
+  var argv = yargs(process.argv.slice(2)).argv;
```

Comment:
To find flags specified by command-line arguments. `opts` in `index.js` has all flags possible.